internal/provider/harness_container_resource_test.go

```diff
@@ -130,16 +130,16 @@ resource "imagetest_harness_container" "test" {
 
 resource "imagetest_feature" "test" {
   name = "Simple container based test"
-  description = "Test that we can spin up a container and run some steps with a working directory"
+  description = "Test that we can spin up a volume and write some stuff in it"
   harness = imagetest_harness_container.test
   steps = [
     {
-      workdir = "/tmp"
+      workdir = "/volume"
```

Comment:
any fun reason for this? it's completely fine I'm just curious

Reply:
This test was meant to verify the volume but due to ✨ copypasta ✨ it ended up like this 🤦🏻‍♀️